.github/workflows/oracle_server_cd_develop.yml

```
@@ -0,0 +1,105 @@
+          key: ${{ secrets.ORACLE_INSTANCE_DEV_PRIVATE_KEY }}
+          port: ${{ secrets.ORACLE_INSTANCE_DEV_PORT }}
+          script: |
+            docker pull ${{ secrets.DOCKER_IMAGE_DEV }}:latest
+
+            export USERNAME=${{ secrets.ORACLE_INSTANCE_USER }}
+            export DOCKER_APP_IMAGE=${{ secrets.DOCKER_IMAGE_DEV }}:latest
+            sudo chmod +x /home/${{ secrets.ORACLE_INSTANCE_USER }}/deploy.sh
+            sudo -E /home/${{ secrets.ORACLE_INSTANCE_USER }}/deploy.sh
+
+            docker image prune -af
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)